test/functional/core/job_spec.lua

```diff
@@ -622,6 +622,27 @@ describe('jobs', function()
       msg = (msg[2] == 'stdout') and next_msg() or msg  -- Skip stdout, if any.
       eq({'notification', 'exit', {0, 42}}, msg)
     end)
+
+    it('jobstart() does not keep ptmx file descriptor open', function()
+      if helpers.pending_win32(pending) then return end
+      -- Start another job (using libuv)
+      nvim('command', 'let g:job_opts.pty = 0')
+      local other_jobid = eval("jobstart(['cat', '-'], g:job_opts)")
+      local other_pid = eval('jobpid(' .. other_jobid .. ')')
+
+      -- Other job doesn't block first job from recieving SIGHUP on jobclose()
+      nvim('command', 'call jobclose(j)')
+      -- Have to wait so that the SIGHUP can be processed by tty-test on time.
+      -- Can't wait for the next message as if this test is a failing case
```

Comment:
The wording here is a little awkward.  Maybe change `as if this test is a failing case` to `because if this test fails`.